velox/functions/prestosql/tests/ArrayContainsTest.cpp

```diff
@@ -26,14 +27,24 @@ using namespace facebook::velox::functions::test;
 
 namespace {
 
-class ArrayContainsTest : public FunctionBaseTest {};
-
-TEST_F(ArrayContainsTest, integerNoNulls) {
-  auto arrayVector = makeArrayVector<int64_t>(
-      {{1, 2, 3, 4}, {3, 4, 5}, {}, {5, 6, 7, 8, 9}, {7}, {10, 9, 8, 7}});
+class ArrayContainsTest : public FunctionBaseTest {
+ public:
+  void testContains(
+      ArrayVectorPtr arrayVector,
+      std::vector<int64_t> search,
```

Comment:
nit: ditto and remove the move, i found that simpler to follow.

Reply:
Fixed